### DISCUSSION TITLE
Tutorial For Sodium Compatible Version

### DISCUSSION THREAD

**Discussion by BlunderG** [IrisShaders/Iris#145] [Q&A] [answered]
Hi! How are you doing? Never mind, let me go straight to the point:

I know you have a beta version that works with Sodium. However, I could not understand how to do it. Can you or someone else make a complete tutorial for who don't understand how do it?

PS: I realized that I didn't know how to download _a thing_ from Github. If you are (or someone else) going to make a tutorial, can you add this as well?

Happy weeks, years and centuries!

**Comment by coderbot16** [comment, marked as answer]
Some documentation has been provided here: https://github.com/IrisShaders/sodium-fabric/blob/1.16.x/iris/README.md

I'm converting this to a discussion since it's not a bug or crash with Iris.
